.github/workflows/tox.yml

```diff
@@ -225,6 +225,7 @@ jobs:
       with:
         password: ${{ secrets.testpypi_password }}
         repository_url: https://test.pypi.org/legacy/
+        skip_existing: true
```

Comment:
My impression is that this is only hiding a mistake we made configuring what triggers a publish job. We already test the package building as part of the "linters" group so we should trigger "publish" only on tags and pushes. tags would go pypi and non-tags to testpypi. This logic should avoid needing a skip.

I am inclined to avoid skip as it hides such logic issues, and it may take some time to discover them. What do you think?

Reply:
tagged releases should go to both PyPI and TestPyPI